### PR TITLE
Fix gitlab/github token warning

### PR DESCRIPTION
**Pull request by ilaner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/48254#issuecomment-1121049728
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-442

## Description
Dont print warning about gitlab/github warning
Fixes issue when getting ReadTimeOut exception

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

**Review by dorschw — APPROVED** [review]

**Review by dorschw — CHANGES_REQUESTED** [review]
```
Validating Packs/DevSecOps/Playbooks/DevSecOps_-_Fetch_PR_-_Triage.yml as playbook
2022-05-09T12:35:50Z [DEBUG]: Could not access GitHub api in `_search_github_repo`
2022-05-09T12:35:51Z [DEBUG]: Could not access GitHub api in `_search_github_repo`
If your repo is in private github repo, configure `DEMISTO_SDK_GITHUB_TOKEN` environment variable or configure `DEMISTO_SDK_REPO_HOSTNAME` environment variable
Could not find the repository name on gitlab - defaulting to demisto/content
Validating deprecated file: Packs/DeprecatedContent/Integrations/integration-Shodan.yml
2022-05-09T12:35:51Z [DEBUG]: Could not access GitHub api in `_search_github_repo`
2022-05-09T12:35:51Z [DEBUG]: Could not access GitHub api in `_search_github_repo`
2022-05-09T12:35:51Z [DEBUG]: Could not access GitHub api in `_search_github_repo`
If your repo is in private github repo, configure `DEMISTO_SDK_GITHUB_TOKEN` environment variable or configure `DEMISTO_SDK_REPO_HOSTNAME` environment variable
Could not find the repository name on gitlab - defaulting to demisto/content
```

has the issue been fixed?

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 * Added HelloWorldSlim template option for *--template* flag in **demisto-sdk init** command.
 * Fixed an issue where the HelloWorldSlim template in **demisto-sdk init** command had an integration id that was conflicting with HelloWorld integration id.
 * Updated the SDK to use demisto-py 3.1.6, allowing use of a proxy with an environment variable.
+* Removed a false warning about configuring the `GITLAB_TOKEN` environment variable when it's not needed.
```
Comment (dorschw) [inline CHANGELOG.md:22]: please merge from master, list seems outdated

File: demisto_sdk/commands/common/tests/git_config_test.py [hunk 2]
```diff
@@ -213,7 +213,7 @@ def test_search_gitlab_id_valid(self, mocker, requests_mock):
         requests_mock.get(url, json=gitlab_response)
         mocker.patch.object(Repo, 'remote', return_value=Urls(['https://code.pan.run/xsoar/content-internal-dist.git']))
         git_config = GitContentConfig()
-        assert git_config.gitlab_id == 3606
+        assert git_config.project_id == 3606
```
Comment (dorschw) [inline demisto_sdk/commands/common/tests/git_config_test.py:216]: please add a comment explaining what this number is and why we test it

Comment (ilaner) [reply demisto_sdk/commands/common/tests/git_config_test.py]: added

File: CHANGELOG.md [hunk 3]
```diff
@@ -18,6 +18,7 @@
 * Added HelloWorldSlim template option for *--template* flag in **demisto-sdk init** command.
 * Fixed an issue where the HelloWorldSlim template in **demisto-sdk init** command had an integration id that was conflicting with HelloWorld integration id.
 * Updated the SDK to use demisto-py 3.1.6, allowing use of a proxy with an environment variable.
+* Removed a warning about configuring GITLAB_TOKEN when running any `demisto-sdk` command.
```
[the remark this answers is not in the document]
Comment (ilaner) [reply CHANGELOG.md]: moved to unreleased

File: CHANGELOG.md (hunk 1 above)
Comment (ilaner) [reply CHANGELOG.md]: moved to unreleased

**Review by dorschw — APPROVED** [review]

**Pull request merged**